net-mail/poppassd_ceti/poppassd_ceti-1.8.5-r3.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils toolchain-funcs pam portability
+
+MY_PN="poppassd"
+MY_P="${MY_PN}-${PV}"
+S=${WORKDIR}/${MY_P}
+
+DESCRIPTION="Password change daemon with PAM support"
+HOMEPAGE="http://echelon.pl/pubs/poppassd.html"
+SRC_URI="http://echelon.pl/pubs/${MY_P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+xinetd"
+
+PATCHES=(
+	${FILESDIR}/nullCheckFix.patch
+	${FILESDIR}/doc-sudoAlternative.patch
+)
+
+DEPEND="sys-libs/pam"
+
+RDEPEND="${DEPEND}
+	xinetd? ( sys-apps/xinetd )
+"
+
+src_compile() {
+	$(tc-getCC) -c ${CFLAGS} ${MY_PN}.c || die "Compile failed."
+	$(tc-getCC) ${LDFLAGS} -o poppassd ${MY_PN}.o -lpam $(dlopen_lib) || die "Linking failed."
+}
```

Comment:
Could you check whether this needs AR modified as well? See qmail.eclass or qmail-notify for example.

Reply:
AR shouldn't be needed:  Unlike your qmail examples (which are apparently modifying config files associated with the built-in makefiles/scripts?), this is bypassing the Makefile completely and running the two commands needed to build it directly.  It only has one .c file, and does not build any libraries.

This is the way -r2 works, but I could potentially change it patch the variables in the Makefile instead of bypassing the Makefile.  That would probably be more complicated, but also more flexible (if a user wanted to add more source files using epatch_user).  Is it worth it?